src/components/Garden/ModalFlows/CreateProposalScreens/CreateProposalScreens.tsx

```diff
@@ -33,7 +33,7 @@ function GoToProposal() {
     async function getProposalId() {
       const id = await extractProposalId(ethers, txHash, 'conviction')
 
-      if (mounted) {
+      if (mounted()) {
```

Comment:
Why removed the parenteses? Its look still a method defined on the hooks.

Btw, its possible rename the var to isMounted instead? if it change to boolean?

Reply:
I added the parênteses since it’s a function, we could just use useEffect, but didn’t want to change more the code. 

I don’t know how it’s working right now 🤷🏼‍♂️

Reply:
Oh! i read inverse?. Make sense